requirements.txt

```diff
@@ -4,3 +4,7 @@ tqdm
 more-itertools
 transformers>=4.19.0
 ffmpeg-python==0.2.0
+click
+pyaudio
+SpeechRecognition
+pydub
```

Comment:
These packages are only needed when running the demo and should be declared as extra in `setup.py ` and excluded from `install_requires`.
(The current ``setup.py`` make all packages in requirements.txt `install_requires`.)